lib/plugins/aws/package/compile/events/apiGateway/lib/method/index.test.js

```diff
@@ -520,6 +520,89 @@ describe('#compileMethods()', () => {
     });
   });
 
+  it('should add request parameter mapped value when explicitly defined', () => {
```

Comment:
Let's test it with help of `runServerless` util, see: https://github.com/serverless/serverless/blob/master/tests/README.md#unit-tests

Reply:
There were no existing `runServerless` tests for the API Gateway. I added a new `apiGateway` fixture and created a separate `fixtures.test.js` file in the same directory as the existing API Gateway tests. I believe it could be confusing to mix and match the existing tests with this different style of testing and it is out of scope of this PR to rewrite all unit tests for API Gateway in the `runServerless` format. 

Please let me know if further changes are required.

Reply:
> I believe it could be confusing to mix and match the existing tests with this different style of testing and it is out of scope of this PR to rewrite all unit tests for API Gateway in the runServerless format.

Yes, definitely I do not expect that. We already have history of such test added to files which have tests written in old format. Then best pattern is to put them into second `describe`, as e.g. here: https://github.com/serverless/serverless/blob/73107822945a878abbdebe2309e8e9d87cc2858a/lib/plugins/aws/package/compile/functions/index.test.js#L2624